pkgs/games/gzdoom/default.nix

```diff
@@ -0,0 +1,33 @@
+{stdenv, fetchFromGitHub, cmake, fmod, mesa, SDL2}:
+
+stdenv.mkDerivation {
+  name = "gzdoom-git";
```

Comment:
SCM packages should have the date of the pinned commit as version number. In this case, that's `2015-05-07`.